docs-new/docs/graphql/core/migrations/config-v2/advanced/auto-apply-migrations.mdx

```diff
@@ -1,17 +1,18 @@
-<div class="meta"
-description="Auto-apply migrations and metadata when the server starts"
-keywords="hasura, docs, auto-apply, migration, metadata, server">
-
-</div>
+---
+sidebar_label: Auto-apply migrations/metadata (config v2)
```

Comment:
minor mismatch in sidebar label

`Auto-apply migrations/metadata when the server starts (config v2)`